app/uk/gov/hmrc/thirdpartyapplication/controllers/AuthorisationWrapper.scala

```diff
@@ -44,6 +46,19 @@ trait AuthorisationWrapper {
     Action andThen AuthenticatedAction()
   }
 
+  def checkOnlyStandardAndSandbox(applicationId: UUID): ActionBuilder[Request] = {
+    Action andThen ApplicationFilter(applicationId)
+  }
+
+    private case class ApplicationFilter(applicationId: UUID) extends AuthenticationFilter() {
```

Comment:
maybe this one could be a bit more generic, taking in a set of access types and a set of environments and "checkOnlyStandardAndSandbox" would populate the right values when creating an instance of this class

Reply:
Andy has made quite a few changes (in a branch) on blocking based on capability and permissions. Maybe check with him before looking at making it more generic.